### DISCUSSION TITLE
Wrong column header?

### DISCUSSION THREAD

**Discussion by R1BLH** [wavelog/wavelog#503] [General]
In the Overview logbook view, the column containing an operator callsign has the header "Station".
![изображение](https://github.com/wavelog/wavelog/assets/16667173/8db11725-78aa-473b-b9f3-17fa19c58ab0)

Same situation in the Advanced Search Results table: the column containing an operator callsign has the header "Station".
![изображение](https://github.com/wavelog/wavelog/assets/16667173/bbf5fbe5-c48d-4c45-b498-a2f67b62d9fa)

**Comment by HB9HIL** [comment]
I don't see a bug here. 

<img width="132" alt="image" src="https://github.com/wavelog/wavelog/assets/80885850/941a8479-5124-46eb-a32f-11897b62984d">
This screenshot is from the advanced search results, the same is shown in the Overview Logbook. 

HB9HIL/M is the callsign which is set as Station Callsign in the Stationlocation. This is correct. The "operator callsign" would be just HB9HIL

**Reply by R1BLH** [reply]
Yes, but using "station" is misleading. Because this word translates to "station location" while the callsign is displayed.
I think it would be better to call this column either "Callsign" or "Station Callsign".

![изображение](https://github.com/wavelog/wavelog/assets/16667173/bfc479bf-ff07-4dbe-a3ee-654da873a2b0)
This is translated view. "Профиль QTH" is translation of "Station location".
